### DISCUSSION TITLE
Add a section as vaults

### DISCUSSION THREAD

**Discussion by AuthorShin** [ghostfolio/ghostfolio#2180] [Ideas]
Thank you for your efforts in building and maintaining this fantastic project.
I have a suggestion: would it be possible to introduce a 'vaults' section? This section could allow users to add multiple vaults, each representing different types of storage such as a wallet, cold storage, safe, bank deposit, exchange balance, etc. These vaults would enable us to manage and monitor our net worth, as well as the locations where we store our money and valuable assets.

Additionally, it would be beneficial to have an option to make transactions or add them as logs with detailed information and dates. For instance, I would like to know why I transferred 12 BTC from my cold storage to my exchange on 12/05/2021, and at what rate.

It would also be helpful to see the real-time value of the assets held in each vault, as well as the total value of all assets based on current market rates. This feature could also include the dates when the assets were added to the vaults.

I believe these enhancements would greatly improve the functionality and user experience of the project.

**Comment by dtslvr** [comment]
Hi @AuthorShin

Thank you very much for your feedback.

What do you think would be the advantage of vaults over accounts? You can already add different activities to an account, check the total value and even attach notes.

**Reply by bptrgx** [reply]
What I think is interesting to keep from this idea of vault is the ability to tag accounts (depending on what type of storage it is, or on anything else) and then be able to filter portfolio pages on these tags.

**Reply by dtslvr** [reply]
I agree, tagging on account level would be a great addition. But before we start with it, I would prefer to realize the tag management (identical to the platform management: _Admin Control_ > _Settings_). Are you interested in working on it, @bptrgx?
